### ISSUE TITLE
gi vs pgi On Ubuntu 19.04

### ISSUE THREAD

**Issue by pulltab** [dlenski/gp-saml-gui#7]
Using python3 (3.7.7) on Ubuntu 19.04 the system was unable to find the `gi` package even after install `python3-gi`.  

I was able to get the script working by installing pgi via `pip3 install pgi` and making the following changes:

```
 #!/usr/bin/env python3
 
-import gi
+import pgi
 import argparse
 import pprint
 import urllib
@@ -15,9 +15,9 @@ from sys import stderr, platform
 from binascii import a2b_base64, b2a_base64
 from urllib.parse import urlparse, urlencode
 
-gi.require_version('Gtk', '3.0')
-gi.require_version('WebKit2', '4.0')
-from gi.repository import Gtk, WebKit2, GLib
+pgi.require_version('Gtk', '3.0')
+pgi.require_version('WebKit2', '4.0')
+from pgi.repository import Gtk, WebKit2, GLib
```

As a more permanent fix, how would we feel about aliasing `pgi as gi` if the `pgi` package is installed?  If pgi is not present then attempt to `import gi`?

**Comment by dlenski** [comment]
Yeah, this is confusing. It appears that there are two different projects providing GObject introspection for Python3: https://github.com/pygobject/pgi (unmaintained, `import pgi`) and https://pygobject.readthedocs.io/en/latest/ (maintained, `import gi`).

> As a more permanent fix, how would we feel about aliasing `pgi as gi` if the `pgi` package is installed? If pgi is not present then attempt to `import gi`?

Good idea.

**Comment by dlenski** [comment]
Closed with #11.
